package.json

```diff
@@ -13,43 +13,47 @@
     "node": ">= 8.0",
     "npm": ">= 6.1"
   },
-  "main": "index.js",
+  "main": "src/index.js",
   "files": [
     "src",
     "assets"
   ],
   "scripts": {
-    "sass": "node ./scripts/sass",
-    "start": "http-server -s -p 3000",
-    "dev": "concurrently -k \"http-server -s -p 3001\" \"webpack -w \" ",
-    "test": "ava src/**/test/**.js"
+    "lint:scss": "stylelint ./assets/scss/**/*.scss --syntax scss --ignore-path .gitignore",
+    "build:scss": "postcss ./assets/scss/clab-pattern.scss --dir ./assets/css/ --ext min.css --verbose",
+    "watch:scss": "npm run build:scss -- --map --watch",
+    "dev-server": "webpack-dev-server --config ./config/webpack.dev.js --port 3000",
```

Comment:
@giotramu if you leave the `webpack.config.js` file in the project's root you don't have to specify the config file path here as command argument.

Reply:
Done.